package.json

```diff
@@ -24,6 +24,7 @@
     "concurrently": "^7.6.0",
     "connect-mongo": "^4.6.0",
     "daisyui": "^2.45.0",
+    "date-and-time": "^2.4.2",

```

Comment:
I'm curious as to how you chose this library. I had mentioned `date-fns`, which to me is something of the de facto standard. It's up to you which dependencies you choose, but usually I'd recommend going with the most popular libraries when you add dependencies. It makes it more likely that a developer coming into your project will have used the library before, and most importantly, the more people that use a library, generally means it's going to be better maintained and have a better featureset.

`date-and-time` has 800,000 weekly npm downloads, whereas `date-fns` has 15 million.

This is also maybe just be being opinionated because I've used `date-fns` a lot and I really like it haha.

Reply:
I chose `date-and-time` library because I came across it a while ago and was trying to use it on my frontend without success. So I really want to try it again this time. Also, it's light-weight (6.87 MB vs 323 kB), not sure if it matters for the scale of my app. On top of everything, I skimmed through their documentation and found what I need quickly.